database/things/fortran.pldb

```diff
@@ -211,6 +211,7 @@ indeedJobs fortran engineer
  2017 447
 linkedInSkill fortran
  2018 148352
+annualReports https://fortran-lang.org/en/news/ officialBlogUrl
```

Comment:
Hi Kimera (@rkimera94),

Why is there "officialBlogUrl" at the end of this line? Should it be on a separate line?

Kind Regards,
Liam